TrafficCapture/dockerSolution/src/main/docker/docker-compose.yml

```diff
@@ -91,32 +89,67 @@ services:
     networks:
       - migrations
     ports:
-      - "9220:9220"
-    healthcheck:
-      test: "lsof -i -P -n"
-    volumes:
-      - ${TRAFFIC_COMPARATOR_DIRECTORY}:/trafficComparator
-      - sharedComparatorSqlResults:/shared
-    command: /bin/sh -c "cd trafficComparator && pip3 install --editable . && nc -v -l -p 9220 | tee /dev/stderr | trafficcomparator -vv stream | trafficcomparator dump-to-sqlite --db /shared/comparisons.db"
+      - "39200:9200"
 
-  jupyter-notebook:
-    image: 'migrations/jupyter_notebook:latest'
+  opensearch-dashboards:
+    image: migrations/opensearch_dashboards:latest # Make sure the version of opensearch-dashboards matches the version of opensearch installed on other nodes
+    container_name: opensearch-dashboards
+    ports:
+      - "5601:5601" # Map host port 5601 to container port 5601
+    expose:
+      - "5601" # Expose port 5601 for web access to OpenSearch Dashboards
     networks:
       - migrations
-    ports:
-      - "8888:8888"
     volumes:
-      - ${TRAFFIC_COMPARATOR_DIRECTORY}:/trafficComparator
-      - sharedComparatorSqlResults:/shared
-    environment:
-      # this needs to match the output db that traffic_comparator writes to
-      - COMPARISONS_DB_LOCATION=/shared/comparisons.db
-    command: /bin/sh -c 'cd trafficComparator && pip3 install --editable ".[data]" && jupyter notebook --ip=0.0.0.0 --port=8888 --no-browser --allow-root'
+      - ./opensearchDashboards/opensearch_dashboards.yml:/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml
+    depends_on:
+      - opensearchanalytics
+
+  otel-collector:
+    build: ./otelcol/
+    image: migrations/otelcol:latest
+    container_name: otel-collector
+    ports:
+      - "4137:4137"
+      - "13133:13133"
+    networks:
+      - migrations
+    depends_on:
+      - opensearchanalytics
+
+  # trafficcomparator:
+  #   image: 'migrations/traffic_comparator:latest'
+  #   networks:
+  #     - migrations
+  #   ports:
+  #     - "9220:9220"
+  #   healthcheck:
+  #     test: "lsof -i -P -n"
+  #   volumes:
+  #     - ${TRAFFIC_COMPARATOR_DIRECTORY}:/trafficComparator
+  #     - sharedComparatorSqlResults:/shared
+  #   command: /bin/sh -c "cd trafficComparator && pip3 install --editable . && nc -v -l -p 9220 | tee /dev/stderr | trafficcomparator -vv stream | trafficcomparator dump-to-sqlite --db /shared/comparisons.db"
+
+  # jupyter-notebook:
+  #   image: 'migrations/jupyter_notebook:latest'
+  #   networks:
+  #     - migrations
+  #   ports:
+  #     - "8888:8888"
+  #   volumes:
+  #     - ${TRAFFIC_COMPARATOR_DIRECTORY}:/trafficComparator
+  #     - sharedComparatorSqlResults:/shared
+  #   environment:
+  #     # this needs to match the output db that traffic_comparator writes to
+  #     - COMPARISONS_DB_LOCATION=/shared/comparisons.db
+  #   command: /bin/sh -c 'cd trafficComparator && pip3 install --editable ".[data]" && jupyter notebook --ip=0.0.0.0 --port=8888 --no-browser --allow-root'
```

Comment:
Yeah, good point. I'm going to uncomment them for now, because I think they deserve their own commit (or ideally a way to conditionally enable them) but they're somewhat annoying for testing something unrelated at this point. My computer can only handle so many containers at once.